connectors/email/src/main/java/io/camunda/connector/email/client/jakarta/outbound/JakartaEmailActionExecutor.java

```diff
@@ -301,16 +301,25 @@ private Multipart getMultipart(SmtpSendEmail smtpSendEmail) throws MessagingExce
     Multipart multipart = new MimeMultipart();
     switch (smtpSendEmail.contentType()) {
       case PLAIN -> {
+        if (smtpSendEmail.body() == null) {
+          throw new MessagingException("Text Body is null");
+        }
         MimeBodyPart textPart = new MimeBodyPart();
         textPart.setText(smtpSendEmail.body(), StandardCharsets.UTF_8.name());
         multipart.addBodyPart(textPart);
       }
       case HTML -> {
+        if (smtpSendEmail.htmlBody() == null) {
+          throw new MessagingException("HTML Body is null");
+        }
         MimeBodyPart htmlPart = new MimeBodyPart();
         htmlPart.setContent(smtpSendEmail.htmlBody(), JakartaUtils.HTML_CHARSET);
         multipart.addBodyPart(htmlPart);
       }
       case MULTIPART -> {
+        if (smtpSendEmail.htmlBody() == null || smtpSendEmail.body() == null) {
+          throw new MessagingException("Text or HTML Body is null");
+        }
```

Comment:
Hey @itsnuyen, instead of throwing a `MessagingException`, could you implement a custom validation method in the model that verifies that these values are not null?

Reply:
Hi @sbuettner, 
I have one questions. Do you mean adapting inside the `SmtpSendEmail` record the validation like this?
```java
switch (contentType) {
  case PLAIN:
    if (body == null) {
      throw new IllegalArgumentException("Email body is null");
    }
  case HTML:
    if (htmlBody == null) {
      throw new IllegalArgumentException("Email html body is null");
    }
  case MULTIPART:
    if (body == null || htmlBody == null) {
      throw new IllegalArgumentException("Email body or html body cannot be null");
    }
}
```
This could also running but as a error message it could be confusing for the end user:
![image](https://github.com/user-attachments/assets/d1658e80-6f66-48e1-b0d5-d1427a144290)
As it state that an exception is raised and it is a JSON_PROCESSING_ERROR

Do you have an Idea how I could provide the end user proper feedback? 
Or do I even misunderstand your request?

Reply:
Hey @itsnuyen I was thinking about something like this as part of the `SmtpSendEmail` record:
```
@AssertTrue(message = "Please provide a proper message body")
        private boolean isValidMessageBody() {
                return body != null || htmlBody != null;
        }
```